esp/src/eclwatch/SFDetailsWidget.js

```diff
@@ -106,14 +110,19 @@ define([
             }
         },
         _onCopyOk: function (event) {
-            this.logicalFile.copy({
-                request: domForm.toObject(this.id + "CopyDialog")
-            });
-            registry.byId(this.id + "CopyDropDown").closeDropDown();
-        },
-        _onCopyCancel: function (event) {
-            registry.byId(this.id + "CopyDropDown").closeDropDown();
+            if (this.copyForm.validate()) {
+                var context = this;
+                this.logicalFile.copy({
+                    request: domForm.toObject(this.id + "CopyForm")
+                }).then(function (response) {
+                    context._handleResponse("CopyResponse.result", response);
+                });
+                registry.byId(this.id + "CopyDropDown").closeDropDown();
+            }
         },
+        // _onCopyCancel: function (event) {
+        //     registry.byId(this.id + "CopyDropDown").closeDropDown();
+        // },
```

Comment:
Remove commented code (is CopyCancel needed?)

Reply:
no, it's not, was meant to be removed.